### DISCUSSION TITLE
incompatibility?

### DISCUSSION THREAD

**Discussion by Kuro100** [Mydayyy/Valheim-ServerSideMap#63] [Q&A]
Is there any known mod incompatibility?

**Comment by Mydayyy** [comment]
Greetings,

Thank you for your question.

Currently the only incompatibility that I know of is crossplay/playfab. So far I have not received any reports of other incompatibilities, but I am sure there are mods out there which don't work in combination with this.

If you discover any incompatible mods Id love to hear about that so I can start a list.

Best Regards
Mydayyy
